silq/parameters/acquisition_parameters_composite.py

```diff
@@ -0,0 +1,526 @@
+
+    Parameters:
+        ESR (dict): `ESRPulseSequence` generator settings for ESR. Settings are:
+            ``stage_pulse``, ``ESR_pulse``, ``ESR_pulses``, ``pulse_delay``,
+            ``read_pulse``.
+        EPR (dict): `ESRPulseSequence` generator settings for EPR.
+            This is optional and can be toggled in ``EPR['enabled']``.
+            If disabled, contrast is not calculated.
+            Settings are: ``enabled``, ``pulses``.
+        pre_pulses (List[Pulse]): Pulses to place at the start of the sequence.
+        post_pulses (List[Pulse]): Pulses to place at the end of the sequence.
+        pulse_sequence (PulseSequence): Pulse sequence used for acquisition.
+        samples (int): Number of acquisition samples
+        results (dict): Results obtained after analysis of traces.
+        t_skip (float): initial part of read trace to ignore for measuring
+            blips. Useful if there is a voltage spike at the start, which could
+            otherwise be measured as a ``blip``. Retrieved from
+            ``silq.config.properties.t_skip``.
+        t_read (float): duration of read trace to include for measuring blips.
+            Useful if latter half of read pulse is used for initialization.
+            Retrieved from ``silq.config.properties.t_read``.
+        min_filter_proportion (float): Minimum number of read traces needed in
+            which the voltage starts low (loaded donor). Otherwise, most results
+            are set to zero. Retrieved from
+            ``silq.config.properties.min_filter_proportion``.
+        traces (dict): Acquisition traces segmented by pulse and acquisition
+            label
+        silent (bool): Print results after acquisition
+        continuous (bool): If True, instruments keep running after acquisition.
+            Useful if stopping/starting instruments takes a considerable amount
+            of time.
+        properties_attrs (List[str]): Attributes to match with
+            ``silq.config.properties``.
+            See notes below for more info.
+        save_traces (bool): Save acquired traces to disk.
+            If the acquisition has been part of a measurement, the traces are
+            stored in a subfolder of the corresponding data set.
+            Otherwise, a new dataset is created.
+        dataset (DataSet): Traces DataSet
+        base_folder (str): Base folder in which to save traces. If not specified,
+            and acquisition is part of a measurement, the base folder is the
+            folder of the measurement data set. Otherwise, the base folder is
+            the default data folder
+        subfolder (str): Subfolder within the base folder to save traces.
+
+    Notes:
+        - All pulse settings are copies of
+          ``ESRParameter.pulse_sequence.pulse_settings``.
+        - For given pulse settings, ``ESRParameter.pulse_sequence.generate``
+          will recreate the pulse sequence from settings.
+    """
```

Comment:
docstring seems to be old, I guess you could also just copy from new pulse sequences docstrings....